src/Blog/BlogPlaceholder.js

```diff
@@ -0,0 +1,66 @@
+                    id="Shape"
+                  />
+                </g>
+              </g>
+              <g id="sake" fill="#FFFFFF" fillRule="nonzero">
+                <path
+                  d="M43.3846154,35 L3.61538462,35 C1.61704103,35 0,36.7145222 0,38.8333333 L0,61.8333333 C0,72.4013222 8.10978974,81 18.0769231,81 L28.9230769,81 C38.8902103,81 47,72.4013222 47,61.8333333 L47,38.8333333 C47,36.7145222 45.382959,35 43.3846154,35 Z"
+                  id="Shape"
+                />
+                <path
+                  d="M34.1669148,12.0413505 C31.0328513,10.4574126 31.0328513,6.99530264 34.1669148,5.41136474 C35.966071,4.50653228 36.4512139,2.67146585 35.2531353,1.3155642 C34.0512347,-0.0432239793 31.6252657,-0.406927373 29.8299314,0.495210991 C22.0577088,4.40454138 22.0556703,13.0472116 29.8299314,16.9575042 C32.9718938,18.5414421 32.9588989,22.0100948 29.8299314,23.5874899 C28.0307751,24.4923224 27.5456323,26.3245023 28.7437109,27.6832905 C29.9382222,29.0366905 32.3585856,29.4115551 34.1669148,28.5036437 C41.9391374,24.5943133 41.9495843,15.9558766 34.1669148,12.0413505 Z"
+                  id="Steam"
+                />
+                <path
+                  d="M17.1669148,12.0413505 C14.0328513,10.4574126 14.0328513,6.99530264 17.1669148,5.41136474 C18.966071,4.50653228 19.4512139,2.67146585 18.2531353,1.3155642 C17.0550567,-0.0432239793 14.6214437,-0.406927373 12.8299314,0.495210991 C5.05770876,4.40454138 5.05567035,13.0472116 12.8299314,16.9575042 C15.9718938,18.5414421 15.9588989,22.0100948 12.8299314,23.5874899 C11.0307751,24.4923224 10.5456323,26.3245023 11.7437109,27.6832905 C12.9382222,29.0366905 15.3585856,29.4115551 17.1669148,28.5036437 C24.9391374,24.5943133 24.9495843,15.9558766 17.1669148,12.0413505 Z"
+                  id="Steam"
+                />
+              </g>
+            </g>
+          </g>
+        </svg>
```

Comment:
I would put all of this svg into a seperate file or component and then import it. Will make this component a lot cleaner